dpppt-backend-sdk/dpppt-backend-sdk-ws/src/test/java/org/dpppt/backend/sdk/ws/controller/GaenControllerTest.java

```diff
@@ -996,75 +977,92 @@ public void testEtag() throws Exception {
 		var now = UTCInstant.now();
 		var midnight = now.atStartOfDay();
 
-		insertNKeysPerDayInInterval(14,
-				midnight.minusDays(4),
-				midnight, midnight.minusDays(1));
+		insertNKeysPerDay(midnight, 14, 10, midnight.minusDays(1), false);
+		insertNKeysPerDay(midnight, 14, 10, midnight.minusHours(12), false);
 
-		insertNKeysPerDayInInterval(14,
-				midnight.minusDays(4),
-				midnight, midnight.minusHours(12));
 		// request the keys with date date 1 day ago. no publish until.
 		MockHttpServletResponse response = mockMvc
-				.perform(get("/v1/gaen/exposed/"
-						+ midnight.minusDays(8).getTimestamp())
-								.header("User-Agent", "MockMVC"))
+				.perform(
```

Comment:
that is really a strange formatting - can't i go on the same line?

Same below

Reply:
Please give a reason if you think the comment is not useful.

Reply:
sorry, i missed that one